phys/module_physics_init.F

```diff
@@ -4168,6 +4175,11 @@ SUBROUTINE mp_init(RAINNC,SNOWNC,GRAUPELNC,config_flags,restart,warm_rain,
         RAINNC(i,j) = 0.
         SNOWNC(i,j) = 0.
         GRAUPELNC(i,j) = 0.
+#if ( EM_CORE==1)
```

Comment:
@BarryHLynn 
Barry,
Why are these values set to zero again? Is there logic covering different cases?